### PR TITLE
Display payment type details on giving history/statement

### PR DESCRIPTION
**Pull request by briancavanaugh**

### CODE REVIEW

File: src/environments/environment.int.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 export const environment: any = {
-  production: false, 
+  production: false,
   crdsEnv: 'int',
-  authRedirectUrl: 'https://int.crossroads.net/signin', 
-  cookieDomain: '.crossroads.net', 
-  apiServiceFinanceEndpoint: 'https://api-int.crossroads.net/finance/api',
+  authRedirectUrl: 'https://int.crossroads.net/signin',
+  cookieDomain: '.crossroads.net',
+  apiServiceFinanceEndpoint: 'http://localhost:52691/api',
```
Comment (jpc11623) [inline src/environments/environment.int.ts]: Set back to the int endpoint

Comment (briancavanaugh) [reply src/environments/environment.int.ts]: Whoops....thought I removed that from my commit.

File: src/app/giving/history/history-detail/history-detail.component.ts [hunk 2]
```diff
@@ -9,10 +17,84 @@ import { GivingHistory } from '../../../models/giving-history';
...
+          break;
+        }
+        case 'bank': {
+          this.paymentDetailsText =
+            'Bank account ending in ' + this.historyRecord.accountNumber;
+          this.faDetailIcon = faUniversity;
+          break;
+        }
+        case 'non-cash/asset': {
+          //TODO: Display notes field
```
Comment (timmyg) [inline src/app/giving/history/history-detail/history-detail.component.ts]: does this todo need to be here still?

File: src/app/giving/history/history-detail/history-detail.component.ts [hunk 3]
```diff
@@ -9,10 +17,84 @@ import { GivingHistory } from '../../../models/giving-history';
...
+          this.faDetailIcon = null;
+          break;
+        }
+        case 'cash': {
+          this.paymentDetailsText = 'Cash';
+          this.faDetailIcon = faMoneyBill;
+          break;
+        }
+        case 'check': {
+          //TODO: Display itemNumber field as check number
```
Comment (timmyg) [inline src/app/giving/history/history-detail/history-detail.component.ts]: todo need to be here?

**Pull request merged**